cmd/cloud-controller-manager/main.go

```diff
@@ -66,8 +67,9 @@ func main() {
 
 	// add controllers disabled by default
 	app.ControllersDisabledByDefault.Insert("gkenetworkparamset")
-
-	command := app.NewCloudControllerManagerCommand(ccmOptions, cloudInitializer, controllerInitializers, fss, wait.NeverStop)
+	aliasMap := names.CCMControllerAliases()
+	aliasMap["nodeipam"] = kcmnames.NodeIpamController
+	command := app.NewCloudControllerManagerCommand(ccmOptions, cloudInitializer, controllerInitializers, aliasMap, fss, wait.NeverStop)
```

Comment:
Do we need `aliasMap["gkenetworkparamset"] = "gkenetworkparamset"`? AFAIK We don't have presubmit test "gkenetworkparamset" controller to validate that on presubmit level.

Reply:
The alias is only required if the name of the controller changes. This controller's name has not changed.